plugins/org.jkiss.dbeaver.ui.app.standalone/src/org/jkiss/dbeaver/ui/app/standalone/DBeaverCommandLine.java

```diff
@@ -52,6 +55,7 @@
 
     public static final String PARAM_HELP = "help";
     public static final String PARAM_FILE = "f";
+    public static final String PARAM_CONFIGURATION_FILE = "conf";
```

Comment:
It's a bad abbreviation either. Take a look at other applications that support a similar feature.